wait-for-it.sh

```diff
@@ -0,0 +1,177 @@
+#!/usr/bin/env bash
```

Comment:
In the comments can you add a link to the source for this file, licensing information, and the reason why this is the right way to do it (https://docs.docker.com/compose/startup-order/)